database/migrations/2021_01_21_170224_create_notification_controllers_table.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+class CreateNotificationControllersTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('notification_controllers', function (Blueprint $table) {
```

Comment:
Laravel convention (which we've used throughout the rest of the project) would name this `controller_position_notification` - would be good to stick to it

Reply:
71ac40a